client/src/entrypoints/admin/core.js

```diff
@@ -264,12 +264,12 @@ $(() => {
 
     // auto focus on search box
     $input.trigger('focus');
+    const newQuery = $input.val();
 
     // eslint-disable-next-line func-names
     const search = function () {
-      const newQuery = $input.val();
       // eslint-disable-next-line @typescript-eslint/no-use-before-define
-      const currentQuery = getURLParam('q');
+      const currentQuery = searchParams.get('q');
```

Comment:
If you replace these first three lines with something like

```
      const newQuery = $input.val();
      const searchParams = new URLSearchParams(window.location.search);
      const currentQuery = URLSearchParams.get('q');
```

then remove `const searchParams = new URLSearchParams(window.location.search);` from below (line 281)

You should be pretty close I think.

Reply:
This should work. Since the value for q is already been set. Let me test and push the changes